tfhe/src/high_level_api/global_state.rs

```diff
@@ -11,7 +11,7 @@ use crate::high_level_api::keys::ServerKey;
 /// (eg a web server that processes multiple requests in multiple threads).
 /// The user however, has to initialize the internal keys each time it starts a thread.
 thread_local! {
-    static INTERNAL_KEYS: RefCell<ServerKey> = RefCell::new(ServerKey::default());
+    static INTERNAL_KEYS: RefCell<Option<ServerKey>> = RefCell::new(None);
```

Comment:
What's the impact of this change?

Reply:
There is no impact (either performance wise or api wise).

That option used to be a bit deeper inside the `ServerKey` struct, it was in `ServerKey::integer_key::key`
Each level of API wrapped had an `Option<ServerKey>`.

Now that the API only uses integers, the `ServerKey` always has an inner integer server key.

But we still need an option as the thread_local storage cannot be instanciated with a key until the user calls `set_server_key`